pybotvac/account.py

```diff
@@ -111,9 +127,9 @@ def refresh_robots(self):
                                        secret=robot['secret_key'],
                                        traits=robot['traits'],
                                        endpoint=robot['nucleo_url']))
-            except requests.exceptions.HTTPError:
-                print ("Your '{}' robot is offline.".format(robot['name']))
-                continue
+            except (requests.exceptions.ConnectionError,
```

Comment:
Do we really want to raise an exception here? That would change the behaviour of pybotvac.

```suggestion
            except requests.exceptions.HTTPError:
                print ("Your '{}' robot is offline.".format(robot['name']))
                continue
            except requests.exceptions.ConnectionError:
                raise NeatoRobotException("Unable to get robot")
```

Reply:
Ah good catch, not sure why I changed this.  Will revert this part.

Reply:
Not sure, but I think there is a difference between Httperror, which means the robot is offline, and connectionerror. As I said, I'm not sure, but I think the second one means that there is an issue with the connection, eg. the network is broken. So the httperror results in ‘continue‘ and the connection error raises an exception.

Reply:
@dshokouhi as we can see here:
https://2.python-requests.org/en/master/_modules/requests/exceptions/

The connection error is parent to the classes proxyerror and sslerror, that are basically network related. I think we should raise an error at anytime when a connection error is thrown. What do you think?

Reply:
Are you saying to raise separate exceptions?  Because as of now we are raising an exception for both errors whenever `raise_for_status()` fails.  I have now corrected this try/catch block to your suggestion.

Reply:
No, I don't think so. Personally I think that we have enough types of errors at the moment.